scripts/src/main/resources/scripts/command/ide

```diff
@@ -561,4 +561,4 @@ else
   doFail "Missing arguments. Call 'devon help ide' to get help."
 fi
 echo
-doSuccess "Completed ${*}"
+doSuccess "Completed ${*}"
```

Comment:
no need to change `ide` commandlet in this PR. Same problem as above phantom commits.